.evergreen/config.in.yml

```diff
@@ -861,17 +861,24 @@ functions:
   "download and merge coverage":
     - command: shell.exec
       params:
-        silent: true
         working_dir: "src"
         script: |
           ${PREPARE_SHELL}
           export AWS_ACCESS_KEY_ID=${aws_key}
           export AWS_SECRET_ACCESS_KEY=${aws_secret}
-          # Download all the task coverage files.
-          # TODO NODE-3897 - finish this function.  the code below this point is untested because
-          #   aws s3 cp fails due to permissions errors
-          aws s3 cp --recursive s3://mciuploads/mongo-node-driver/${revision}/${version_id}/ coverage/
 
+          sudo dnf install --assumeyes awscli
```

Comment:
This is required until the rhel8 evergreen images have the aws cli installed.

Reply:
There is no timeline for them to look at my PR yet so I kept this here in order to move the PR along.

Reply:
Looks like the work is in-progress - https://jira.mongodb.org/browse/BUILD-16116

I think we should wait on the build ticket before merging this PR